README.md

```diff
@@ -11,14 +11,19 @@ The tests depend on having Firefox (default) or Chrome downloaded.
 1. `bundle install`
 1. `bundle exec rake webdrivers:geckodriver:update`
 
-See the `Configuration` section below for instructions on using Chrome and on tweaking default browser settings.
+See the `Authentication Configuration` section below for
+- authentication token instructions (dor_services_app, dor_services, etd)
```

Comment:
I'm not sure what the parenthetical is providing here.

Reply:
it indicates that there are 3 diff authentication tokens listed.   I missed "dor_services".